### PR TITLE
[EA Forum only] only show post page digest ad after a few reads

### PR DESCRIPTION
**Pull request by s-cheng**
The [previous version of the post page digest ad](https://github.com/ForumMagnum/ForumMagnum/pull/8740) got us [a few new subscribers per day](https://app.hex.tech/9e4abdcd-b18c-49ca-acfc-3ec5249ddb1a/app/b7b415b6-758a-4747-a851-0e1f565614ac/latest), but most people who saw it dismissed or ignored it. Based on the A/B test, users who saw the ad did not bounce at a higher rate, so it seems worth keeping.

For now, we are going to try only showing the ad after the user has viewed a few posts, so that we're more likely to show it to receptive users. I've also closed the A/B test, so now we are showing it on all clients.

And previously we did not show the ad on very small screens, but now that you need to read multiple posts before seeing the ad, I've enabled it on all screen sizes:

Screen Shot 2024-03-14 at 5 25 55 PM

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206846813740749) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/abTests.ts [hunk 1]
```diff
@@ -230,37 +230,3 @@ export const offerToAddCalendlyLink = new ABTest({
...
-    control: {
-      description: "Show the description that our team wrote",
-      weight: 1,
-    },
-    '80k': {
-      description: "Show the description copied from the 80K job board",
-      weight: 1,
-    },
-  },
-});
```
Comment (s-cheng) [inline packages/lesswrong/lib/abTests.ts:266]: Removed both of these tests that are now closed

**Comment by s-cheng** [comment]
@Will-Howard I guess I'm curious if you think there is no version of this that is worth keeping, or you think that we haven't found the version worth keeping yet, and if it's the latter what do you think it should look like? My baseline view is that getting more digest subscribers is quite good because it's a low-effort way to stay engaged with the Forum, and I would bet that a digest subscriber is similar in value to an MAU.

**Comment by Will-Howard** [comment]
@s-cheng 

> How would you propose we measure that?

It's unlikely that we would be able to detect the negative impact, based on the rule of thumb that measuring less than a 10% change isn't generally possible with the data we have (from [this](https://app.hex.tech/9e4abdcd-b18c-49ca-acfc-3ec5249ddb1a/app/ec7b7486-54cb-4700-9be8-7d844b762c7d/latest), and eyeballing the [time on site](https://app.hex.tech/9e4abdcd-b18c-49ca-acfc-3ec5249ddb1a/app/b7b415b6-758a-4747-a851-0e1f565614ac/latest?selectedStaticCellId=9c9e70d3-4bca-426e-a30c-eab8aac22f32) A/B test data).

I think the A/B test was a bit of a foregone conclusion here because the benefit side has no noise (even 1 sign up is a positive signal), whereas the cost side has enough noise that we couldn't detect the size of effect that could plausibly result from the change.

In the absence of data I think it's reasonable on common sense grounds to think that having the ad makes the overall experience maybe 1% worse, for someone who doesn't click on it. Some reasons going into this:
- I personally find ads like these a bit annoying
- The average time per session is about 100s, and having the ad there adds about 1s of closing it and/or minor distraction

Another way you could come at this is to think about whether we would be ok with having 10 things about as distracting as this ad in order to get to a 1.3% sign up rate. (Sidenote: I tried to look up what a typical sign up rate is but I couldn't find a good direct comparison, a "good" clickthrough rate for banner ads is apparently 0.5-1% though).

> I would bet that a digest subscriber is similar in value to an MAU

Hm I would guess it's a bit lower (maybe 10x lower) but this seems like a reasonable ballpark. Looking at the dashboard again now I am actually a bit more impressed by the absolute numbers. We get about 20 total signups/day otherwise so this would be a 10-20% increase.
Screenshot 2024-03-18 at 10 01 11

...

Sorry this isn't very conclusive, I'll leave the final decision up to you

**Comment by s-cheng** [comment]
@Will-Howard Appreciate the response! I'm curious what you think of the version that requires 5 post reads before appearing. I agree with your analysis, so I admit that perhaps I should not have deployed the original version. But take for example my personal experience with substack: their subscription ad is way more intrusive than ours, I find it a bit annoying, but I feel like I've gotten more value from it than harm because I like that it's easy to subscribe to an author. So I think it's not obvious that it me that it's worse, although I think you're likely right. But I think the version that appears after 5 post reads mitigates most of your concern? Or do you think this version is still net negative, like for example because you object to the UX rather than the fact that it appears on the page?

**Comment by Will-Howard** [comment]
@s-cheng Honestly looking at the absolute numbers updated me to thinking it was ~neutral overall (i.e. that it actually is having a big effect on digest subscribers), so I think with the 5 post reads check it's probably worth it 👍

**Pull request merged**